frontend/src/app/components/about/about.component.html

```diff
@@ -406,8 +406,15 @@ <h3 i18n="about.maintainers">Project Maintainers</h3>
   <div class="copyright">
     <div class="title">
       Copyright &copy; 2019-2023<br>
-      The Mempool Open Source Project
+      Mempool Space K.K.<br>
+      and other shadowy super-coders
     </div>
+    <p>
+      The Mempool Open Source Project&reg;, Mempool Accelerator&trade;, Mempool Enterprise&reg;, Mempool Liquidity&trade;, mempool.space&reg;, the mempool logo;, the mempool Square logo;, the mempool Blocks logo;, the mempool.space Vertical Logo;, and the mempool.space Horizontal logo are either registered trademarks or trademarks of Mempool Space K.K in Japan, the United States, and/or other countries.
+    </p>
+    <p>
+      See our <a href="https://mempool.space/trademark-policy">Trademark Policy and Guidelines</a> for more details, published on &lt;https://mempool.space/trademark-policy&gt;.
+    </p>
```

Comment:
This is the copyright notice, trademark notice goes in the trademark notice section below :)